docs/resources/Sticker.md

```diff
@@ -0,0 +1,137 @@
+# Sticker Resource
+
+### Sticker Object
+
+Represents a sticker that can be sent in messages. The URL for fetching sticker assets is currently private.
+
+###### Sticker Structure
+
+| Field       | Type                                            | Description                                                                                                                                                  |
+| ----------- | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| id          | snowflake                                       | id of the sticker                                                                                                                                            |
+| pack_id?    | snowflake                                       | for standard stickers, id of the pack the sticker is from                                                                                                    |
+| name        | string                                          | name of the sticker                                                                                                                                          |
+| description | string                                          | description of the sticker                                                                                                                                   |
+| tags        | string                                          | for guild stickers, the name of a unicode emoji representing the sticker's expression. for standard stickers, a comma-separated list of related expressions. |
```

Comment:
I feel like there should be a mention that the name of the unicode emoji is of Discord's emoji set, rather than the official unicode name.